--- a/src/dynapi/SDL_dynapi.h
+++ b/src/dynapi/SDL_dynapi.h
@@ -35,9 +35,6 @@
    updated SDL can transparently take advantage of them, but your program will
    not without this feature. Think hard before turning it off.
 */
-#ifdef SDL_DYNAMIC_API // Tried to force it on the command line?
-#error Nope, you have to edit this file to force this off.
-#endif
 
 #ifdef SDL_PLATFORM_APPLE
 #include "TargetConditionals.h"
